src/loaddb/load_db.c

```diff
@@ -1157,7 +1157,9 @@ get_loaddb_args (UTIL_ARG_MAP * arg_map, load_args * args)
   args->compare_storage_order = utility_get_option_bool_value (arg_map, LOAD_COMPARE_STORAGE_ORDER_S);
   args->table_name = table_name ? table_name : empty;
   args->ignore_class_file = ignore_class_file ? ignore_class_file : empty;
-  args->no_user_specified_name = utility_get_option_bool_value (arg_map, LOAD_NO_USER_SPECIFIED_NAME_S);
+  /* set to true for testing. When the test is complete, the option should be checked. (by youngjinj) */
+  // args->no_user_specified_name = utility_get_option_bool_value (arg_map, LOAD_NO_USER_SPECIFIED_NAME_S);
+  args->no_user_specified_name = true;
```

Comment:
Is the test not complete?
Is it correct to reflect the code like this?

Reply:
Shell testing is still in progress in QA.

Previously, the system parameter PRM_ID_NO_USER_SPECIFIED_NAME was set to true.
However, changing system parameters affects utilities other than loaddb.
So change it like this.

When the shell test is finished in QA, it will be changed to receive the option value again.